omod/src/main/webapp/resources/scripts/components/LabTrackingOrderDetailsController.js

```diff
@@ -175,8 +178,13 @@ angular.module("labTrackingOrderDetailsController", [])
             scope: {
                 order: '=',
             },
+            controller: function($scope) {
+              $scope.isObject = function (obj) {
+                return typeof obj === 'object';
```

Comment:
I'd recommend

```suggestion
                return obj !== null && typeof obj === 'object';
```

Reply:
Makes sense... I have to change this in a couple different places, so not committing this yet to remind me of this.

Reply:
Changed, in conjunction with the "format diagnosis" function that Mike mentioned below